ckanext/ontario_theme/templates/internal/organization/read.html

```diff
@@ -21,6 +21,9 @@
     (_('Last Modified'), 'metadata_modified desc'),
     (_('Popular'), 'views_recent desc') if g.tracking_enabled else (false, false) ]
   %}
+  {% if not sort_by_selected %}
+    {% set sort_by_selected = 'metadata_modified desc' %}
```

Comment:
It's being sorted by last modified but the order by drop down is still defaulting to display "Relevance"